javascript-sdk/examples/log-into-sites/log-into-sites.js

```diff
@@ -0,0 +1,46 @@
+const { chromium } = require('playwright');
+const { wrap, configure } = require('agentql');
+
+require('dotenv').config();
+
+// Set the URL to the desired website
+const URL = 'https://practicetestautomation.com/practice-test-login/';
+
+const LOGIN_QUERY = `
+{
+    username_field
+    password_field
+    submit_btn
+}
+`;
+
+async function main() {
+  // Configure the AgentQL API key
+  configure({ apiKey: process.env.AGENTQL_API_KEY });
```

Comment:
I recommend removing this part and covering keys in the installation docs, not the sample apps